examples/documentation/documentation.bal

```diff
@@ -20,16 +20,16 @@ public type DummyObject abstract object {
     // This is the documentation attachment for function `doThatOnObject`.
     # `doThatOnObject` is an attached function for the object `DummyObject`.
     #
-    # + paramOne - This is the description for the `doThatOnObject` function
-    #              parameter.
-    # + return - This is the description for the `doThatOnObject` function return
-    #            value.
+    # + paramOne - This is the description for the parameter of
+    #              `doThatOnObject` function.
+    # + return - This is the description for the return value of
+    #            `doThatOnObject` function.
     public function doThatOnObject(string paramOne) returns (boolean);
 };
 
 // This is the documentation attachment for record `DummyType`.
-# `DummyType` is a user defined record type in Ballerina program.
-# This `DummyType` is used to hold a string type data field and an int type
+# `DummyType` is a user defined record type in your Ballerina program.
```

Comment:
Do we need to say `your`? Same for other changes in this PR too.

I don't think we've used `your` in general in BBEs.